e2e/test/register.spec.js

```diff
@@ -1,26 +1,42 @@
 
 describe('Sign Up tests', function(){
-    let elSignUpLink = element(by.xpath('//*[@id="login-form"]/p/a/signupmodal/a'));
+    let elSignUpLink = element(by.xpath('/html/body/app-root/app-home-page/div/div/div/button[2]/signupmodal/a'));

```

Comment:
Can this be improved? A full xpath selector like this will break whenever this or the ancestor elements are moved, making this test very brittle to changes on the source. Same with several other buttons/inputs below.